libs/akita-ng-entity-service/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@datorama/akita-ng-entity-service",
+  "version": "2.0.0",
+  "description": "Akita entity service",
+  "bugs": {
+    "url": "https://github.com/datorama/akita/issues"
+  },
+  "homepage": "https://github.com/datorama/akita/tree/master/libs/akita-ng-entity-service#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/datorama/akita",
+    "directory": "libs/akita-ng-entity-service"
+  },
+  "license": "Apache License 2.0",
+  "author": "Netanel Basal",
+  "keywords": [
+    "angular",
+    "akita"
+  ],
+  "peerDependencies": {
+    "@angular/common": ">= 9 < 10",
```

Comment:
I was perhaps a bit too strict on this one. I believe the current version works with +7.

Also how do you perfer these ranges? Should we just say from vX and let users "discover" when it doesn't work anymore?
Or should we keep it as is `< 10` and let the users know that there are no guarantees. The latter requires a bit more work, but is also the better choice IMO

Reply:
You can remove this. We're not coupled to Angular.

Reply:
that package is

Reply:
Angular plugins coupled to Angular.

Reply:
Yes, and that's the package.json file for `@datorama/akita-ng-entity-service`

Reply:
`>= 7 < 10` have been agreed upon